src/AuthLog/AuthLogService.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @copyright Copyright (c) Wizacha
+ * @license Proprietary
+ */
+declare(strict_types = 1);
+
+namespace Wizaplace\SDK\AuthLog;
+
+use Wizaplace\SDK\AbstractService;
+
+final class AuthLogService extends AbstractService
+{
+    /**
+     * Get AuhtLog
+     *
+     * int $id: AuthLog specific id
+     */
+    public function get(int $id) : array
+    {
+        $this->client->mustBeAuthenticated();
+
+        return $this->client->get(
+            sprintf('/api/v1/security/viewlog/%d', $id)
```

Comment:
Le client est déjà configuré sur `/api/v1/`
```suggestion
            sprintf('security/viewlog/%d', $id)
```